runtime/src/accounts_db.rs

```diff
@@ -169,7 +169,6 @@ impl AccountStorageEntry {
     pub fn new(path: &str, fork_id: Fork, id: usize, file_size: u64) -> Self {
         let p = format!("{}/{}.{}", path, fork_id, id);
         let path = Path::new(&p);
-        let _ignored = remove_dir_all(path);
```

Comment:
This is the main change of the PR. I'm asking why is calling `remove_dir_all` here an issue?

Reply:
apparently it's not the issue, I actually got a the SIGILL on this PR with the "fix" in place...

Reply:
ok good to know. Here is my experiment: https://github.com/solana-labs/solana/pull/5220

hasn't failed yet...